connectors/sources/network_drive.py

```diff
@@ -639,13 +639,29 @@ async def get_entity_permission(self, file_path, file_type):
                 access=DirectoryAccessMask.READ_CONTROL,
             )
         for permission in list_permissions or []:
+            mask = permission["mask"].value
```

Comment:
Can you add some inline comments? As someone who hasn't spent a lot of time with network drives, I'm getting lost in the "mask", "sid", "rid", "ace", verbiage. Some comments to explain what's happening would be helpful.

Reply:
Apologies, you are right, without much idea of network drives, it is a bit complicated to review this PR. I'll update the file with relevant inline comments 👍